ivy/data_classes/array/experimental/linear_algebra.py

```diff
@@ -675,3 +675,29 @@ def tucker(
             tol=tol,
             verbose=verbose,
         )
+
+    def randomized_range_finder(
+        A: Union[ivy.Array, ivy.NativeArray],
+        /,
+        *,
+        n_dims: Optional[int],
+        n_iter: Optional[int] = 2,
+        seed: Optional[int] = None,
+    ):
+        """
+        Compute an orthonormal matrix (Q) whose range approximates the range of A, i.e.,
+        Q Q^H A ≈ A.
+
+        Parameters
+        ----------
+        A : 2D-array
+        n_dims : int, dimension of the returned subspace
```

Comment:
```suggestion
        n_dims
            dimension of the returned subspace
```
We generally write parameter docstring as:
```
<parameter_name>
       Description of the parameter
```

Reply:
Would be nice if you could update all the parameters  in the doctstring to follow this format.